### ISSUE TITLE
Add option to crop "flyback"

### ISSUE THREAD

**Issue by ercius** [OpenChemistry/stempy#237]
The sparse data in the HDF5 file includes the "flyback" in the last column. Add an keyword option to SparseArray.from_hdf5() (default True) to crop the last column which effectively is the same as:
```python
sp = SparseArray.from_hdf5('data_scan5_th4.5_electrons.h5')
sp = sp[: ,:-1, :, :] # remove the fly back column
```

**Comment by ercius** [comment]
working on this in #304

**Comment by ercius** [comment]
fixed in #304
